usage-api/nodejs/index.js

```diff
@@ -0,0 +1,62 @@
+// Get the needed environment variables
+require('dotenv').config();
+let apiKey = process.env.APIKEY;
+let contentType = 'application/json'
+let appId = process.env.APPID;
+let region = process.env.REGION;
+let orgId = process.env.ORGID;
+
+// Function calls to the three usage api
+overallCustomerCount();
+usagePerCustomerWithinRegion();
+individualCustomerSubscription();
+
+let request = require('request');
+
+// Function to get the overall customer count 
+function overallCustomerCount() {
+    let options = {
+        'method': 'GET',
+        'url': 'https://billable-vendor-usage-api.usw2.pure.cloud/v1/regions?appIds=' + appId,
```

Comment:
query params should be encoded using [encodeURIComponent()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURIComponent)

Reply:
updated the variables with encodeURIComponent()